docs/dynamic_batch_size.md

```diff
@@ -0,0 +1,195 @@
+
+- The example uses model [resnet](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/resnet50-binary-0001/README.md).
+
+- While using resnet model with grpc_serving_client.py the script processes the output from the server and displays the inference results using previously prepared file with labels. Inside this file each image has assigned number, which indicates the correct recognition answer.  
+
+## Steps
+Clone OpenVINO&trade; Model Server github repository and enter `model_server` directory.
+#### Download the pretrained model
+Download model files and store it in `models` directory
+```Bash
+mkdir -p models/resnet/1
+curl https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/1/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://download.01.org/opencv/2021/openvinotoolkit/2021.1/open_model_zoo/models_bin/1/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
+```
+
+#### Pull the latest OVMS image from dockerhub
+Pull the latest version of OpenVINO&trade; Model Server from Dockerhub :
+```Bash
+docker pull openvino/model_server:latest
+```
+
+#### OVMS configuration file
+Create new file named `config.json` :
+```json
+{
+   "model_config_list": [
+       {
+           "config": {
+               "name": "resnet",
+               "base_path": "/models/resnet"
+           }
+       }
+   ],
+   "pipeline_config_list": [
+       {
+           "name": "resnet50DAG",
+           "inputs": [
+               "0"
+           ],
+           "demultiply_count" : 0,
+           "nodes": [
+               {
+                   "name": "resnetNode",
+                   "model_name": "resnet",
+                   "type": "DL model",
+                   "inputs": [
+                       {
+                           "0": {
+                               "node_name": "request",
+                               "data_item": "0"
+                           }
+                       }
+                   ],
+                   "outputs": [
+                       {
+                           "data_item": "1463",
+                           "alias": "1463"
+                       }
+                   ]
+               }
+           ],
+           "outputs": [
+               {"1463": {
+                       "node_name": "resnetNode",
+                       "data_item": "1463"}}
+           ]
+       }
+   ]
+}
+```
+
+#### Start ovms docker container with downloaded model
+Start ovms container with image pulled in previous step and mount `models` directory :
+```Bash
+docker run --rm -d -v $(pwd)/models:/models -v $(pwd)/config.json:/config.json -p 9000:9000 openvino/model_server:latest --config_path config.json --port 9000
```

Comment:
add in the command plugin_config setting. Number of streams is particularly important here

Reply:
Correct me if I am wrong, but adding it in the command is not possible due to running ovms with configuration file. However, I am going to add this setting to config.